src/e2e/settings.test.ts

```diff
@@ -11,8 +11,11 @@ describe("Browser - settings", () => {
   afterEach(app.screenshot.bind(app));
   afterAll(app.dispose.bind(app));
 
-  it("shows the settings dialog when the button is clicked", async () => {
```

Comment:
This pre-existing test was broken but we didn't notice as the file name didn't match the pattern the test runner uses. It didn't do anything worthwhile so I've just removed it. It dated to when settings was a top-level menu.